WPFWijkagent/WPFWijkagent/DelictDialog.xaml.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Windows;
+using System.Windows.Controls;
+using WijkagentModels;
+
+namespace WijkagentWPF
+{
+    /// <summary>
+    /// Interaction logic for SocialMediaDialogue.xaml
+    /// </summary>
+    public partial class DelictDialog : Window
+    {
+        private readonly DelictDialogController _controller;
+
+        /// <summary>
+        /// instantiates the window
+        /// </summary>
+        /// <param name="offence">data to be shown</param>
+        public DelictDialog(Offence offence)
+        {
+            InitializeComponent();
+            _controller = new DelictDialogController();
+
+            wpfDelict.DataContext = offence;
+            _controller.DisplayMessages(offence, wpfLVMessages);
+        }
+
+        private void wpfBPrint_Click(object sender, RoutedEventArgs e)
```

Comment:
kan deze niet weg

Reply:
Welk stuk bedoel je hier. De hele klasse, de constructor, de controller, de print functie. Is me niet duidelijk

Reply:
de click functie die niks doet.